include/brainfuck.h

```diff
@@ -16,9 +16,9 @@
 #ifndef BRAINFUCK_H
 #define BRAINFUCK_H
 
-#define BRAINFUCK_VERSION_MAJOR @brainfuck_VERSION_MAJOR@
-#define BRAINFUCK_VERSION_MINOR @brainfuck_VERSION_MINOR@
-#define BRAINFUCK_VERSION_PATCH @brainfuck_VERSION_PATCH@
```

Comment:
Did you remove the auto generated version on purpose? (I actually did remove it in one of my commits by accident because git keeps changing my files, but I think including it would be a good idea)

Reply:
Is there any way to prevent it from rewriting the `include/brainfuck.h` and instead write it to the build folder?

I think the current approach will cause people unintentionally committing these changes into the repository.

Reply:
I'm not really sure what you mean by "writing it to the build folder",  but if you think this code forms a problem go ahead and remove it.

Reply:
Currently, every time you run `cmake`, the `include/brainfuck.h` file is overwritten. I think this will cause problems in the future (like unintentionally committing these changes).

Reply:
Ah okay, I understand now. I didn't know cmake would actually overwrite the file, but that explains why I thought git kept screwing with my `brainfuck.h`. Do remove it for now, I will see if there is a more "flexible" approach for autogenerating versions.